src/shared/codelens/codeLensUtils.ts

```diff
@@ -28,6 +28,13 @@ import { ExtContext } from '../extensions'
 
 export type Language = 'python' | 'javascript' | 'csharp'
 
+interface SamDebugCodeLensInput {
```

Comment:
Why do we need this new interface instead of adding a property to `AddSamDebugConfigurationInput`?

Reply:
Found a way to make it work with just the one updated interface.